waltz-common/src/main/java/com/wepay/waltz/common/message/MessageCodecV0.java

```diff
@@ -7,6 +7,9 @@
 import com.wepay.waltz.common.util.Utils;
 import com.wepay.waltz.exception.RpcException;
 
+import java.util.HashMap;
+import java.util.Map;
+
 public class MessageCodecV0 implements MessageCodec {
```

Comment:
Not necessary to update V0 codec. This is for communicating with old servers.

Reply:
aahh ok. fixed.